modular-docs-manual/content/topics/module_creating-assemblies.adoc

```diff
@@ -10,7 +10,7 @@ An assembly could represent an article, "chapter" in a book, or even an entire b
 . Create an assembly file for each user story that you identified.
 +
 --
-Be sure to follow the conventions for xref:anchor-and-file-names[naming anchors and files]. For example: `guided-decision-tables.adoc`
+Follow the conventions for xref:module_anchor-and-file-names-concept[naming anchors and files]. For example: `assembly_designing-guided-decision-tables.adoc`
```

Comment:
```suggestion
Follow the conventions for xref:anchor-and-file-names-concept[naming anchors and files]. For example: `designing-guided-decision-tables.adoc`
```

Reply:
This looks like it added a prefix where there was none. AFAIK we are implementing the prefix removal for everything including assemblies, but please correct me if I'm wrong!

Reply:
Yes, this is just a (bit unrelated) fix of an xref (in a more convenient way). And regarding the example - we preserve prefixes in file names.